pkg/app/pipedv1/plugin/kubernetes/deployment/server.go

```diff
@@ -51,13 +52,18 @@ type applier interface {
 	ForceReplaceManifest(ctx context.Context, manifest provider.Manifest) error
 }
 
+type logPersister interface {
+	StageLogPersister(deploymentID, stageID string) logpersister.StageLogPersister
+}
+
 type DeploymentService struct {
 	deployment.UnimplementedDeploymentServiceServer
 
 	RegexPool    *regexpool.Pool
 	Logger       *zap.Logger
 	ToolRegistry toolRegistry
 	Loader       loader
+	LogPersister logPersister
```

Comment:
I do not actually remember the reason why these properties should be set as public 🤔

Reply:
I think it's a mistake, so I'll make these private on another PR.